pkg/oc/lib/graph/genericgraph/test/runtimeobject_nodebuilder.go

```diff
@@ -129,40 +126,36 @@ func BuildGraph(path string) (osgraph.Graph, []runtime.Object, error) {
 	g := osgraph.New()
 	objs := []runtime.Object{}
 
-	abspath, err := filepath.Abs(path)
+	data, err := ioutil.ReadFile(path)
 	if err != nil {
 		return g, objs, err
 	}
-
-	builder := resource.NewFakeBuilder(
-		func(version schema.GroupVersion) (resource.RESTClient, error) {
-			return &fake.RESTClient{}, nil
-		},
-		func() (meta.RESTMapper, error) {
-			return testrestmapper.TestOnlyStaticRESTMapper(ocscheme.ReadingInternalScheme), nil
-		},
-		func() (restmapper.CategoryExpander, error) {
-			return resource.FakeCategoryExpander, nil
-		})
-
-	r := builder.
-		WithScheme(ocscheme.ReadingInternalScheme).
-		FilenameParam(false, &resource.FilenameOptions{Recursive: false, Filenames: []string{abspath}}).
-		Flatten().
-		Do()
-
-	if r.Err() != nil {
-		return g, objs, r.Err()
-	}
-
-	infos, err := r.Infos()
+	scheme := runtime.NewScheme()
+	kubernetesscheme.AddToScheme(scheme)
+	api.Install(scheme)
+	legacy.InstallExternalLegacyAll(scheme)
+	codecs := serializer.NewCodecFactory(scheme)
+	decoder := codecs.UniversalDeserializer()
+	obj, err := runtime.Decode(decoder, data)
```

Comment:
@deads2k for the tests that rely on scheme I've decided to create a standalone scheme installing only the APIs I cared about, wdyt?

Reply:
> @deads2k for the tests that rely on scheme I've decided to create a standalone scheme installing only the APIs I cared about, wdyt?

That's a good idea.  You may wan to to use `k8s.io/apimachinery/pkg/api/testing.SchemeForOrDie` which allows you to just list the various install methods.